internal/pkg/externalplugins/cherrypicker/cherrypicker_test.go

```diff
@@ -388,9 +401,16 @@ func testCherryPickIC(clients localgit.Clients, t *testing.T) {
 	if err := s.handleIssueComment(logrus.NewEntry(logrus.StandardLogger()), ic); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	got := prToString(ghc.prs[0])
-	if got != expected {
-		t.Errorf("Expected (%d):\n%s\nGot (%d):\n%+v\n", len(expected), expected, len(got), got)
+	if len(ghc.prs) != len(expectedBranches) {
+		t.Fatalf("Expected %d PRs, got %d", len(expectedBranches), len(ghc.prs))
+	}
+
+	for i, expectedBranch := range expectedBranches {
+		got := prToString(ghc.prs[i])
+		expected := expectedFn(expectedBranch)
```

Comment:
Can we integrate `prToString` and several `expectedFn` functions into one function to convert the PR structure into a string?

Reply:
I don't think it's appropriate, because on the one hand they don't mean the same thing (although the format is similar), and on the other hand the expectedFn method uses some closures to get some current variables. Also their assignees are different. So I think we can keep it the same. Unifying them doesn't bring much benefit.